test/mocha.opts

```diff
@@ -2,6 +2,7 @@ test/helpers/*.js
 test
 --compilers js:babel-core/register
 --require ignore-styles
+--require babel-polyfill
```

Comment:
I really, really want to use the `async`/`await` style as because (as you mentioned) it is supported in node 7.6. The inclusion of the `--compilers js:babel-core/register` flag doesn't quite perform transpilation beyond some stylistic rewriting as necessary to make the imports work for the client side code, but it doesn't modify the global parameters.

That flag was ported over simply because the tests required them. I would for the moment, unfortunately, recommend removing async/await until we can find a way to tell babel to only apply to the client mocha tests instead of all tests, hence the problem you're facing.

I didn't realize that this would be a deeper issue when I saw async/await and would have recommended removing until we can rectify.

Sorry for the confusion!